hw29/hw29/Network.swift

```diff
@@ -0,0 +1,21 @@
+
+import Foundation
+
+class Network {
+    enum APIHandler {
+        static let todos = "https://jsonplaceholder.typicode.com/todos"
+    }
+    
+    func loadData(completion: @escaping ([Todos]) -> Void) {
```

Comment:
Красота. Еще можно сдесь было сделать, для обработки ошибок
```suggestion
    func loadData(completion: @escaping (Result<[Todos], Error>) -> Void) {
```